joomla/joomla-cms
# Cleanup installer plugins

#### Summary of Changes

This change does a few things:
- Split the URL/Folder installation plugin into two separate plugins
- Removed obsolete files
- Removed obsolete code
- Cleanup other issues (thank you @andrepereiradasilva taken also from #10296)
- Added a message if no plugin is enabled
- Moved the language strings to the plugin language files
#### Testing Instructions for a clean installation
1. Download the zip file from this pull request containing the complete Joomla installation
2. Do a clean installation
3. Check the tabs are still under Manage -> Install
#### Testing Instructions for an update
1. Install Joomla 3.5.1
2. Download the zip file from this pull request containing the complete Joomla installation
3. Setup a local update server
4. Update the 3.5.1 to 3.6.0 using the local update server
5. Check the tabs are still under Manage -> Install
### Testing Instructions for both
1. Disable all installer plugins in Plugin Manager
2. Go to Manage -> Install and check there is a message saying there are no plugins enabled
3. Check the link in the message works
